client/extensions/woocommerce/index.js

```diff
@@ -112,6 +112,12 @@ const getStorePages = () => {
 			path: '/store/reviews/:site',
 		},
 		{
+			container: Reviews,
+			configKey: 'woocommerce/extension-reviews',
+			documentTitle: translate( 'Reviews' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **13** times:
`translate( 'Latest Reviews' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).